iconrpcserver/icon_rpcserver_app.py

```diff
@@ -157,25 +154,19 @@ def run_in_foreground(conf: 'IconConfig'):
         certfile = ssl_context[0]
         keyfile = ssl_context[1]
 
-    options = {
-        'bind': f"{host}:{conf[ConfigKey.PORT]}",
-        'workers': conf[ConfigKey.GUNICORN_WORKER_COUNT],
-        'worker_class': "sanic.worker.GunicornWorker",
+    options = conf.get(ConfigKey.GUNICORN_CONFIG, {})
+    options.update({
+        'bind': f"{conf[ConfigKey.HOST]}:{conf[ConfigKey.PORT]}",
         'certfile': certfile,
-        'SERVER_SOFTWARE': gunicorn.SERVER_SOFTWARE,
         'keyfile': keyfile,
+        'SERVER_SOFTWARE': gunicorn.SERVER_SOFTWARE,
         'capture_output': False
-    }
-
-    # TODO : update gunicorn settings at once
-    if conf.get(ConfigKey.GUNICORN_WORKER_TMP_DIR, None):
-        options.update({'worker_tmp_dir': conf[ConfigKey.GUNICORN_WORKER_TMP_DIR]})
+    })
```

Comment:
From changes that you made in another file, I think you prefer double quotation, rather than single one.

How about change single quotations at key names?

Reply:
It has not unified rule. I applied majority quotation rule in similar group.